src/tools/jest/testUtils.tsx

```diff
@@ -0,0 +1,35 @@
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import * as React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {render} from '@testing-library/react-native';
+import {QueryClient, QueryClientProvider} from '@tanstack/react-query';
+
+export const testQueryClient = new QueryClient({
+  defaultOptions: {queries: {retry: false}},
+});
+const Stack = createNativeStackNavigator();
+
+function renderScreen(
+  ui: React.ReactElement,
+  screenName: string,
+  initialParams?: Partial<object | undefined>,
+  {...options} = {},
+) {
+  function Wrapper({children}: {children: () => React.ReactElement}) {
```

Comment:
This type should work better. You may not need to use type assertion to make types pass `WeatherScreen as unknown as React.ReactElement`.
```suggestion
  function Wrapper({children}: {children: () => React.ReactNode}) {
```

Reply:
@jaworek I'm getting the following warning

![Screenshot 2023-07-13 at 17 19 08](https://github.com/callstack-internal/onboarding-adrian/assets/137059906/af0be767-e224-4c58-a0bb-e87f3e1275ed)

Reply:
You can resolve by making argument `ui` have type `() => React.ReactNode`, instead of `React.ReactNode` or when using `renderScreen` you can do `renderScreen(<WeatherScreen />, Screen.Weather);`